src/Cake.Common.Tests/Fixtures/Tools/MSBuildRunnerFixture.cs

```diff
@@ -30,6 +30,8 @@ public MSBuildRunnerFixture(bool is64BitOperativeSystem)
             KnownMSBuildPaths.Add("/Program86/MSBuild/12.0/Bin/amd64/MSBuild.exe");
             KnownMSBuildPaths.Add("/Program86/MSBuild/14.0/Bin/MSBuild.exe");
             KnownMSBuildPaths.Add("/Program86/MSBuild/14.0/Bin/amd64/MSBuild.exe");
+            KnownMSBuildPaths.Add("/Program86/Microsoft Visual Studio/2017/Enterprise/MSBuild/15.0/Bin/MSBuild.exe");
```

Comment:
I think that we should add some tests for community and professional as well here.